cmd/influx/user_resource_mapping.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/cmd/influx/internal"
+)
+
+func membersAddF(urm influxdb.UserResourceMapping, flgs Flags) error {
+	mappingSvc, err := newUserResourceMappingService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize members service client: %v", err)
+	}
+
+	if err = mappingSvc.CreateUserResourceMapping(context.Background(), &urm); err != nil {
+		return fmt.Errorf("failed to add member: %v", err)
+	}
+	writeUserResourceMapping(urm)
+	return nil
+}
+
+func membersRemoveF(resourceID, userID influxdb.ID, flgs Flags) error {
+	mappingSvc, err := newUserResourceMappingService(flags)
```

Comment:
The arg passed in is `flgs`, not `flags`. Maybe change lines 12, 13 and 25 to be `flags` instead?

Reply:
the only reason `flags` works here is because it's a global variable :)

Reply:
good find, that's exactly why we don't want to use any global vars